app/models/feature.rb

```diff
@@ -1,18 +1,2 @@
 class Feature
-  def self.enabled?(feature_name, user:)
-    case feature_name.to_sym
-    when :new_developer_fields_banner
-      true
-    when :message_read_indicator
-      true
-    when :pricing_v3
-      true
-    when :pricing_v2
-      true
-    when :reply_via_email
-      true
-    when :obfuscate_developer_urls
-      !Rails.env.production?
```

Comment:
Sorry, I wrote this issue before this feature was added. Can you please keep this case and the code around it?

Reply:
Yup please check if its alright